.github/workflows/acceptance.yml

```diff
@@ -112,6 +113,7 @@ jobs:
           type: frontend
 
   webpack:
+    if: ${{ needs.files-changed.outputs.acceptance == 'true' }}
```

Comment:
some of these use `if ${{ ... }}` and others just use `if: ...` -- the brackets aren't needed I think